db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, body VARCHAR(140), date VARCHAR, username VARCHAR);
```

Comment:
This db instruction creates a `date` column, but in your peep class you push to a `time` column, was it renamed afterward?

Reply:
good spot, i edited this at the end of my project and didn't double check my conventions. whoops!